src/server/status_server.rs

```diff
@@ -20,6 +20,7 @@ use tikv_alloc::error::ProfError;
 use tikv_util::collections::HashMap;
 use tikv_util::metrics::dump;
 use tikv_util::timer::GLOBAL_TIMER_HANDLE;
+use rsperftools;
```

Comment:
Who can tell me why this line is needed ?! Without this line `backtrace-rs` can only give me the first frame (perf_signal_handler)

Reply:
@kennytm

Reply:
The same reason as https://github.com/rust-lang/rust/issues/64402#issuecomment-530852886 I suppose (to link the crate).

Reply:
Even more wield! If I set rsperftools as a `path` dependency, I cannot get full stack backtrace even if I use it/extern crate it explicitly. However, both `git` dependency and normal dependency  work well (with use statement)!